docs/how-tos/creating-an-account-locally.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Creating an Account Locally
+---
+
+Once you've got FxA running locally you'll probably want to create an account that you can use for development and testing purposes. This process can be done almost exactly like you would in production:
+
+1. Visit the [root page](http://localhost:3030/), which should ask you to enter your email. Enter the email you want to use for the account.
+
+  If you're taken to `/signin` you can just click "Use a different account" to return to the email field.
```

Comment:
We could use a note here about or linking to other docs about how to create an account through the "Looking for Firefox Sync?" link.